packages/projection-typeorm/test/sinks/types.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+// Consider moving this to projection package src types
```

Comment:
Seems reasonable to do this

Reply:
760c8990fdbbd5564b9c360a3fd6caed17728a70 and 3e5853d654f3947a68a52f78d7981ecb94de814d